config.toml

```diff
@@ -214,35 +214,24 @@ no = 'Sorry to hear that. Please <a href="https://github.com/kubeflow/website/is
 [params.links]
 # Docsy: "User" links will show on left side of footer and in the community page if you have one.
 [[params.links.user]]
-	name ="Twitter"
+	name = "Twitter"
 	url = "https://twitter.com/kubeflow"
 	icon = "fab fa-twitter"
-        desc = "Follow us on Twitter to get the latest news"
+    desc = "Follow us on Twitter to get the latest news!"
 [[params.links.user]]
-	name = "GitHub"
-	url = "https://github.com/kubeflow/kubeflow"
-	icon = "fab fa-github"
-        desc = "Development takes place here!"
-[[params.links.user]]
-	name = "Slack"
-	url = "https://kubeflow.slack.com/join/shared_invite/zt-cpr020z4-PfcAue_2nw67~iIDy7maAQ"
+	name = "Slack Server"
+	url = "/docs/about/kubeflow-slack-workspace"
 	icon = "fab fa-slack"
-        desc = "Chat with other project contributors"
+    desc = "Join the Kubeflow Slack Workspace!"
 [[params.links.user]]
 	name = "Mailing list"
-	url = "https://groups.google.com/forum/#!forum/kubeflow-discuss"
+	url = "/docs/about/kubeflow-mailing-list"
 	icon = "fa fa-envelope"
-        desc = "Discussion and help from your fellow users"
-[[params.links.user]]
-	name = "Calendar"
```

Comment:
I believe this is still useful as in https://github.com/kubeflow/community/tree/master/calendar. I join Kubeflow community meeting every week using this calendar. Why do we remove this?

Reply:
I have readded it (but made it a link to `/docs/about/community-call/`).

I think that's better than linking directly to it, so we don have to keep two links up to date if/when we change calanders.

Reply:
Sounds good, it makes sense!